lib/preview.js

```diff
@@ -120,15 +124,60 @@ const buildUploadFileToPreviewBufferCallback = (sessionInfo, notifyMessage) => {
       return;
     }
     const destPath = getDesignManagerPath(src, dest, filePath);
-    const uploadPromise = uploadFile(portalId, filePath, destPath);
+    const uploadPromise = uploadFile(accountId, filePath, destPath);
     triggerNotify(notify, notifyMessage, filePath, uploadPromise);
   };
 };
 
 const initialPreviewBufferUpload = async (sessionInfo, filePaths) => {
-  const { portalId, src, dest } = sessionInfo;
-
-  return uploadFolder(portalId, src, dest, fileMapperArgs, {}, filePaths);
+  const { accountId, src, dest } = sessionInfo;
```

Comment:
Updated the initial upload to be less spammy and only draw attention if there's actually a problem - instead display a progress bar & print out any errors at the end

Uploading Screen Recording 2024-02-07 at 3.17.21 PM.mov…